book/src/5.in-depth-guides/4.auth/1.quickstart.md

```diff
@@ -0,0 +1,74 @@
+# Quickstart
+
+In this section we will walk through creating an app that can access the [Microsoft Graph APIs](https://learn.microsoft.com/en-us/graph/overview) on behalf of the user by authenticating them with the [Microsoft Entra ID](https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id) oauth provider. 
+
+> [!Note]
+> It is possible to authenticate the user into [other auth providers](https://learn.microsoft.com/en-us/azure/bot-service/bot-builder-concept-identity-providers?view=azure-bot-service-4.0&tabs=adv2%2Cga2#other-identity-providers) like Facebook, Github, Google, Dropbox, and so on.
+
+> [!Note]
+> This is an advanced guide. It is highly recommended that you are familiar with [creating an app](https://microsoft.github.io/teams-ai/2.getting-started/1.quickstart.html) and [running it in Teams](https://microsoft.github.io/teams-ai/2.getting-started/3.running-in-teams.html) before attempting to follow this guide.
+
+> [!Warning]
+> User authentication does not work with the developer tools setup. You have to run the app in Teams. Follow these [instructions](../../2.getting-started/3.running-in-teams.html#debugging-in-teams) to run your app in Teams.
+
+## Setup Instructions
+
+### Create an app with the `graph` template
```

Comment:
Can you include the `graph` is the default name of the oauth connection, but you can change that by supplying `defaultOauthConnectionName` in `app`?